content/4_Gold/Combinatorics.problems.json

```diff
@@ -60,6 +60,21 @@
       }
     }
   ],
+  "sample6": [
+    {
+      "uniqueId": "erasing-vertices",
```

Comment:
fix the unique id

atcoder ids should be the source followed by the problem name: `ac-ErasingVertices`